redisinsight/api/src/modules/rdi/models/rdi-dry-run.ts

```diff
@@ -0,0 +1,13 @@
+export enum DyRunJobStatus {
+  Success = 'success',
+  Fail = 'fail',
+}
+
+export class RdiDryRunJobResult {
+  status: DyRunJobStatus;
+
+  // TODO validate with RDI team
+  data?: any;
+
+  error?: string;
```

Comment:
not sure if error will be a plain string since we must have some logic on it. But for now it is more then enough

Reply:
Yep, will see what kind of error we will have